Apps/WebClient/src/ClientApp/package.json

```diff
@@ -4,10 +4,10 @@
     "version": "0.2.0",
     "scripts": {
         "serve": "vue-cli-service serve",
-        "build": "vue-cli-service lint --max-warnings 0 --no-fix && vue-cli-service build",
```

Comment:
not part of the scope of this PR

Reply:
This is required to automate the linting of test files. Making changes like https://github.com/bcgov/healthgateway/pull/2255#discussion_r624420983 much easier